### ISSUE TITLE
Develop equitable hiring process & assign someone to hiring committees to implement process

### ISSUE THREAD

**Issue by nclowell** [OARS-SAFS/DEI#49]

**Comment by JLGnotes** [comment]
@nclowell Is this in addition to the SAFS Diversity Hiring Tool handout? And yes, it sounds like an E&I committee rep can be part of the hiring committee. 

Related to issue #53 .

**Comment by nclowell** [comment]
Yep. The hiring tool is a set of definitions and examples of recommended rubric items and criteria for evaluation. In other words, it's a starting point to better hiring. I think there needs to be an agreed upon equitable hiring practice (can be different for staff, faculty, etc.) that comes with a procedure checklist that is followed throughout the hire.
